config.go

```diff
@@ -32,8 +32,10 @@ func getConfig() *types.Configuration {
 		Alertmanager:    types.AlertmanagerOutputConfig{ExtraLabels: make(map[string]string), ExtraAnnotations: make(map[string]string), CustomSeverityMap: make(map[types.PriorityType]string)},
 		CloudEvents:     types.CloudEventsOutputConfig{Extensions: make(map[string]string)},
 		GCP:             types.GcpOutputConfig{PubSub: types.GcpPubSub{CustomAttributes: make(map[string]string)}},
+		OTLP:            types.OTLPOutputConfig{},
 	}
 
+	otlpSetEnvs()
```

Comment:
why did you choose to create a dedicated function and not doing like for the others ouputs?

Reply:
mainly because of the complexity added from `OTEL_*` native SDK env vars vs `OTLP_*` introduced here (+ its auto-config machinery) and the needed ~mapping between them.

Reply:
I see, is there a way to explicitly list all of them and expose them in the config file?

Reply:
Unsure tbh, although I see your point, e.g. ppl wanting to set headers via config rather than being "forced" to set OTEL_EXPORTER_OTLP_TRACES_HEADERS.

Recall I did try to craft above headers under `viper` config machinery, but ended up needing to create a golang type hierarchy at each `_` mapped to e.g. `Otel.Exporter.Otlp.Traces.Headers` equivalent types "nesting":

1) Do you know a way around this (a nested type at each `_` from the env)?
2) Alternatively, we could "shortcut" these the same way as I did with `OTLP_TRACES_ENDPOINT` (own, yaml config supported) <-> `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` or `OTEL_EXPORTER_OTLP_ENDPOINT` (SDK) as e.g.:
   * `OTLP_TRACES_HEADERS` (own) <-> `OTEL_EXPORTER_OTLP_TRACES_HEADERS, OTEL_EXPORTER_OTLP_HEADERS`
   * `OTLP_TRACES_TIMEOUT` (own) <-> _[ditto]_
   * `OTLP_TRACES_PROTOCOL` (own) <-> _[ditto]_

wdyt?